$OpenBSD$

on powerpc and arm, chars default to unsigned, causes an issue here.

--- libr/line/dietline.c.orig	Tue Jul 19 09:35:10 2011
+++ libr/line/dietline.c	Thu Jul 21 02:52:49 2011
@@ -247,7 +247,7 @@ R_API char *r_line_readline() {
 	const char *gcomp_line = "";
 	static int gcomp_idx = 0;
 	static int gcomp = 0;
-	char buf[10];
+	signed char buf[10];
 	int ch, i; /* grep completion */
 
 	I.buffer.index = I.buffer.length = 0;
